libdevice/test/sycl_noinline.txt

```diff
@@ -21,52 +21,52 @@ RUN: llvm-dis %t.bc -o - | FileCheck %s
 
 RUN: clang-offload-bundler -type=o -unbundle \
 RUN:     -targets=sycl-spir64-unknown-unknown \
-RUN:     -input=%libsycldevice_obj_dir/libsycl-fallback-cassert.o -output=%t.bc
+RUN:     -input=%libsycldevice_obj_dir/libsycl-fallback-cassert.old.o -output=%t.bc
```

Comment:
Is it necessary to do this renaming now? why not keep default offloading model used now without additional suffixes and new offloading model to keep using `new` suffixes?